### PR TITLE
Joop party new

### PR DESCRIPTION
**Pull request by JoopStark**
## Description
Feat: View Party new date time and invitation

Test: Sad path testing for User new

## Type of change

- [x] fix
- [x] feat
- [x] test
- [ ] refactor
- [ ] docs

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [ ] thorough testing has been implemented if adding feature
- [x] all tests pass

### CODE REVIEW

**Comment by JoopStark** [comment]
Does need more thorough testing for New Viewing Party

File: app/controllers/users_controller.rb
Comment (reedhillmar) [inline app/controllers/users_controller.rb:1]: good catch

File: app/controllers/viewing_parties_controller.rb
Comment (reedhillmar) [inline app/controllers/viewing_parties_controller.rb:1]: Great job figuring this out. Definitely got a little brain bendy for me last night

File: app/views/viewing_parties/new.html.erb
Comment (reedhillmar) [inline app/views/viewing_parties/new.html.erb:1]: We pair programmed some of this. Great job figuring out check_box_tag

File: spec/features/users/new_spec.rb
Comment (reedhillmar) [inline spec/features/users/new_spec.rb:1]: Hooray for sad pathing! Hooray for working validations!

**Review by reedhillmar — APPROVED** [review]
Nice sad path testing, and good job figuring out the check box after I called it last night!

**Pull request merged**